src/e2e/java/teammates/e2e/pageobjects/StudentProfilePage.java

```diff
@@ -159,12 +159,21 @@ public void showPictureEditor() {
         waitForUploadEditModalVisible();
     }
 
-    public void verifyPhotoSize(String height, String width) {
-        assertEquals(height, browser.driver.findElement(By.className("profile-pic")).getCssValue("height"));
-        assertEquals(width, browser.driver.findElement(By.className("profile-pic")).getCssValue("width"));
+    public void closePictureEditor() {
         click(uploadEditModal.findElement(By.className("close")));
     }
 
+    public void verifyPhotoSize(int height, int width) {
+        String browserHeight = browser.driver.findElement(By.className("profile-pic")).getCssValue("height");
+        String browserWidth = browser.driver.findElement(By.className("profile-pic")).getCssValue("width");
+        //Remove px from Dimension Strings to turn it into a number
+        float imageHeight = Float.parseFloat(browserHeight.replaceFirst("px$", ""));
+        float imageWidth = Float.parseFloat(browserWidth.replaceFirst("px$", ""));
+        assertEquals(height, imageHeight, 1.0);
+        assertEquals(width, imageWidth, 1.0);
+        closePictureEditor();
```

Comment:
A "verify" action should not have any business with closing picture editor. If you need this to be done on all operations, it should be part of the `uploadProfilePicAndVerifyDimensions` method instead.